README.md

```diff
@@ -452,15 +452,19 @@ recommend this resource as a good starting point: https://strongloop.com/strongb
 
 ### walkSync(dir)
 
-Lists all files inside a directory recursively
+Lists all files and directories along with their `fs.stats`, inside a directory recursively.
```

Comment:
agreed. What about put it just like `Lists all files and directories inside a directory recursively.` or `Lists all paths inside a directory recursively.`, since it explains what it returns, right after that.